src/components/inputComponents/inputIngredients.js

```diff
@@ -0,0 +1,11 @@
+export default function InputIngredients() {
```

Comment:
I see you've created input for each component. I understand the intention, but I think you'd better off with one component for your Form instead, and make that component receive an `onSubmit` prop.

Why?
Right now this components are not really reusable

Reply:
Yes, a Form component is the better approach!